### PR TITLE
Add more missing syntax tests

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: test/libyul/yulSyntaxTests/builtin_identifier_6.yul [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+	// Test for the unreachable 6272_error
```
Comment (axic) [inline test/libyul/yulSyntaxTests/builtin_identifier_6.yul:2]: These are unreachable in the current parser, but good test for #9555.

File: test/libsolidity/syntaxTests/inlineAssembly/shadowing/variable_by_opcode.sol [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        uint mload;
+    }
+    function g() public pure {
+        uint mload;
+        assembly {
+        }
+    }
+}
+// ----
+// Warning 8261: (109-119): Variable is shadowed in inline assembly by an instruction of the same name
```
Comment (axic) [inline test/libsolidity/syntaxTests/inlineAssembly/shadowing/variable_by_opcode.sol:12]: This check only executes if there is an assembly block, and it uses `evmasm::c_instructions` and not the dialect.

**Comment by axic** [comment]
The windows failure is some unrelated SMT issue.

File: scripts/error_codes.py [hunk 3]
```diff
@@ -220,15 +220,15 @@ def examine_id_coverage(top_dir, source_id_to_file_names, new_ids_only=False):
...
     old_source_only_ids = {
-        "1123", "1218", "1220", "1584", "1823", "1950",
-        "1988", "2461", "2512", "2592", "2657", "2800", "2842", "2856",
-        "3046", "3263", "3356", "3441", "3682", "3876",
-        "3893", "3996", "4010", "4281", "4802",
-        "5052", "5073", "5170", "5188", "5272", "5347", "5473",
-        "5622", "6041", "6084", "6272", "7110", "7128", "7186",
-        "7589", "7593", "7653", "7812", "7885", "8065", "8084", "8140",
-        "8261", "8312", "8592", "8758", "9011",
-        "9085", "9390", "9440", "9547", "9551", "9615", "9980",
```
Comment (leonardoalt) [inline scripts/error_codes.py:231]: Many of the ids removed here do not have new tests. Am I missing anything?

File: test/libsolidity/syntaxTests/tryCatch/no_special.sol [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        try gasleft() {
+        } catch {
+        }
+        try type(address) {
+        } catch {
+        }
+    }
+}
+// ----
+// TypeError 5347: (72-76): Try can only be used with external function calls and contract creation calls.
```
Comment (chriseth) [inline test/libsolidity/syntaxTests/tryCatch/no_special.sol:15]: The error message seems to be wrong.

Comment (chriseth) [reply test/libsolidity/syntaxTests/tryCatch/no_special.sol:15]: Ah sorry, misread.

**Review by chriseth — APPROVED** [review]

File: scripts/error_codes.py (hunk 3 above)
Comment (leonardoalt) [reply scripts/error_codes.py:231]: @chriseth @axic what about the removed ids?

Comment (axic) [reply scripts/error_codes.py:231]: They were already tested somewhere, just not removed from the list. And some of them were actually not generated anymore.

Comment (leonardoalt) [reply scripts/error_codes.py:231]: Ah ok

**Pull request merged**